### PR TITLE
feat: add dev mode flag

### PR DESCRIPTION
**Pull request by Cafe137**
Usage:

`REACT_APP_DEV_MODE=1 npm start`

### CODE REVIEW

**Comment by agazso** [comment]
If you add the same condition to the `chequebook` as well then it will work with Bee in dev mode.

The only hickup I saw is that the overlay address on the Info page will show `undefined[...]undefined`, see the attached screenshot:
Screenshot 2021-11-16 at 18 19 20

**Comment by Cafe137** [comment]
@agazso Can you please check?

**Comment by agazso** [comment]
> @agazso Can you please check?

works well

File: src/providers/Bee.tsx [hunk 1]
```diff
@@ -113,11 +112,13 @@ function getStatus(
     blockchainConnection: Boolean(nodeAddresses?.ethereum),
     debugApiConnection: Boolean(debugApiHealth?.status === 'ok'),
     apiConnection: apiHealth,
-    topology: Boolean(topology?.connected && topology?.connected > 0),
+    // FIXME: `REACT_APP_DEV_MODE` is a temporary workaround to be able to develop with only one node
```
Comment (vojtechsimetka) [inline src/providers/Bee.tsx:115]: Would you mind adding this to the readme? That also increases the chance we don't forget to remove it :)

**Review by vojtechsimetka — APPROVED** [review]
Thanks, works well!

**Pull request merged**